src/utils/discord.ts

```diff
@@ -0,0 +1,16 @@
+import { Message, MessageCreateOptions } from 'discord.js'
+
+/**
+ * Sends a direct message (DM) to the author of a message if they have enabled DMs.
+ *
+ * @param {Message} message - The original message object.
+ * @param {MessageCreateOptions} payload - The payload of the message to be sent.
+ */
+export async function sendDm(
+  message: Message,
+  payload: MessageCreateOptions,
+): Promise<void> {
+  if (message.author.dmChannel) {
```

Comment:
if the bot has never send the user a DM before, will `dmChannel` exist? do you need to create it in that case?

see: https://www.reddit.com/r/Discordjs/comments/nmhvnx/dmchannel_seems_to_always_be_null/